src/test/java/io/openliberty/tools/intellij/it/SingleModLibertyLSTestCommon.java

```diff
@@ -358,11 +356,6 @@ public static void prepareEnv(String projectPath, String projectName) {
         UIBotTestUtils.validateImportedProjectShowsInLTW(remoteRobot, projectName);
         UIBotTestUtils.closeLibertyToolWindow(remoteRobot);
 
-        // pre-open project tree before attempting to open server.xml
-        ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofMinutes(2));
-        JTreeFixture projTree = projectFrame.getProjectViewJTree(projectName);
-        projTree.expand(projectName, "src", "main", "liberty", "config");
```

Comment:
I thought this block was necessary for setting up @ajm01 's server.xml LS feature tests

Reply:
this is fine to remove here - the same expand logic is in place in the 'openConfigFile' method that is called next to open each individual file needed by bhe testcases.